### PR TITLE
Review common chapter

### PR DESCRIPTION
**Pull request by andreas-junghanns**

### CODE REVIEW

File: docs/2____common_concepts.adoc [hunk 1]
```diff
@@ -19,32 +19,25 @@ To uniquely map between the FMU variables and the network variables, FMU variabl
...
-Input and output variables show if these variables belong to signals or frames that are received or sent, respectively.
+For the low cut, variables represent send and receive bus traffic and their respective clocks.
+Input and output variables show if these variables belong to signals or bus traffic that are received or sent, respectively.
 Input clocks must be triggered and are connected by the importer to their corresponding output clocks of potentially different clock types.
 For example, a periodic output clock indicates that the FMU can only send signals or frames periodically, while a countdown clock indicates that the FMU can send signals at varying time steps.
 
 Using output clocks of different types allows simulations with different timing accuracy and performance.
-These clocks allow the FMU to specify the next send event efficiently and accurately, the importer will trigger these clocks according to these clock specifications as accurately as it can.
+Some clock types allow the FMU to specify the next send event efficiently and accurately, the importer will trigger these clocks according to these clock specifications as accurately as it can.
```
Comment (bmenne-dspace) [inline docs/2____common_concepts.adoc:30]: We need a big hint "You can use trigged Clocks but they may be not your friend". Maybe at another place within the document or using a link and write a chapter into the implementer's guide? See also Triggered clocks: forbidding or discouraging? #14

**Comment by bmenne-dspace** [comment]
While the values and semantic of the clock variables are clear, the binary frame variables are opaque to the importer but have internal structure to implement the transport mechanism of the specific network technology, **see [[FrameVariable]]**(https://modelica.github.io/fmi-ls-bus/main/#FrameVariable). --> [[FrameVariable]] link is broken.

**Comment by bmenne-dspace** [comment]
I am not satisfied with the "System Composition" chapter because it has a fundamental problem: it is in the general part, so it applies to high cut and low cut, but it only describes the architectures on the basis of the low cut. We should consult here again to solve the problem.

File: docs/2____common_concepts.adoc (hunk 1 above)
Comment (andreas-junghanns) [reply docs/2____common_concepts.adoc:30]: Added two sentences to that effect in 2__common_concepts. Please check.

**Comment by bmenne-dspace** [comment]
> While the values and semantic of the clock variables are clear, the binary frame variables are opaque to the importer but have internal structure to implement the transport mechanism of the specific network technology, **see [[FrameVariable]]**(https://modelica.github.io/fmi-ls-bus/main/#FrameVariable). --> [[FrameVariable]] link is broken.

done

**Comment by bmenne-dspace** [comment]
> I am not satisfied with the "System Composition" chapter because it has a fundamental problem: it is in the general part, so it applies to high cut and low cut, but it only describes the architectures on the basis of the low cut. We should consult here again to solve the problem.

Issue added

**Review by bmenne-dspace — APPROVED** [review]

**Pull request merged**